dockers/cemba/vcftoallc/convert-vcf-to-allc.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import argparse
+import gzip
+import re
+
+COMMENT = re.compile(r"^#")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+      "--input_vcf", "-i", dest="input_vcf", default=None, help="input VCF"
+    )
+    parser.add_argument(
+      "--output_allc", "-o", dest="output_allc", default=None,
+      help="output ALLC"
+    )
+    args = parser.parse_args()
+
+    writeBinary = False
+    if args.input_vcf:
+        with gzip.open(args.input_vcf, "rt") if args.input_vcf.endswith(
```

Comment:
This spacing is very hard to read. Maybe keep the `(".gz")` on the same line and start the next line with the else condition

Reply:
Fixed